### ISSUE TITLE
Api key are exposed in the network

### ISSUE THREAD

**Issue by nabilramy** [web3-storage/web3.storage#2277]
Hi there,
Im using your npm library in my project and i found that my environments (your api keys) are in exposed in the car request in the network, how to solve this issue??

**Comment by alanshaw** [comment]
Can you elaborate please? Your API token needs to be sent to authenticate yourself to the service.

**Comment by nabilramy** [comment]
yes I know, but since I'm using it as a third party in my service (in frontend code) to store clients data encrypted I can't reveal my api keys, why there's not tokenized urls or something to protect that
